package.json

```diff
@@ -18,9 +18,11 @@
     "babel-preset-react": "^6.5.0",
     "chai": "^3.5.0",
     "enzyme": "^2.3.0",
-    "eslint": "^2.2.0",
-    "eslint-config-airbnb": "^6.0.2",
-    "eslint-plugin-react": "^4.1.0",
+    "eslint": "^2.9.0",
+    "eslint-config-wagtail": "^0.1.0",
```

Comment:
TODO: move this to wagtail org once it exists. And we need to figure out who gets access to the npm package as well.